spec/operators/defaultIfEmpty-spec.js

```diff
@@ -1,50 +1,42 @@
-/* globals describe, it, expect */
+/* globals describe, it, expect, expectObservable, hot */
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.prototype.defaultIfEmpty()', function () {
-  it('should return the argument if Observable is empty', function (done) {
-    var emptyObservable = Observable.empty();
-    emptyObservable.defaultIfEmpty(2)
-      .subscribe(function (x) {
-        expect(x).toBe(2);
-      }, null, done);
+  var defaultValue = 'x';
```

Comment:
I'd rather keep this var inside each test case, even if it looks like repetition, because it makes tests look more obvious, less magical.